src/document/DocumentAnnotator.scss

```diff
@@ -11,15 +11,41 @@
         pointer-events: none;
     }
 
-    &.ba-is-highlighting {
+    &.ba-discoverability-enabled {
         .textLayer {
-            > span {
-                @include ba-HighlightCreator-cursor;
+            pointer-events: none; // Ensures that click events are received on the create region layer
+
+            span {
+                pointer-events: auto; // Text layer still receives text highlight events
+            }
+        }
+
+        &.ba-is-create--highlight {
+            .textLayer {
+                > span {
+                    @include ba-HighlightCreator-cursor;
+                }
+
+                ::selection {
+                    background-color: rgba($bdl-yellow, .33);
+                }
             }
+        }
```

Comment:
`.ba-is-create--highlight` should not be controlled by discoverability FF, right? Should it live outside of `.ba-discoverability-enabled`?

Reply:
I think it should be controlled by discoverability? I was discussing with @ConradJChan and it looks like feature changes related to discoverability should be nested inside the `.ba-is-discoverability-enabled` class.